### PR TITLE
Extra parameters for built-in bump2d node

### PR DESCRIPTION
**Pull request by luisbarrancos**

### CODE REVIEW

File: src/appleseedmaya/extensionAttributes.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //
 // This software is released under the MIT license.
 //
+// Copyright (c) 2017 Luis Barrancos, The appleseedhq Organization
```
Comment (dictoon) [inline src/appleseedmaya/extensionAttributes.cpp]: There should be a single copyright notice. See https://github.com/appleseedhq/appleseed/wiki/Copyright-Notices for the detailed explanation.

File: scripts/appleseedMaya/aetemplate/__init__.py [hunk 2]
```diff
@@ -59,6 +69,11 @@ def buildBody(self, nodeName):
             self.__buildVisibilitySection()
             self.endLayout()
 
+        if self.thisNode.type() == 'bump2d':
+            self.beginLayout('Appleseed' ,collapse=1)
```
Comment (dictoon) [inline scripts/appleseedMaya/aetemplate/__init__.py]: Insert space after comma.

**Review by dictoon — APPROVED** [review]
Thanks! Reviewed the style, not the actual content as I'm not competent for that. @est77, I'll let you review please!

**Comment by luisbarrancos** [comment]
It should be ok now, With this we can proceed to bump2d in OSL and more testing.

**Review by est77 — APPROVED** [review]

**Pull request merged**